src/Auth0.AuthenticationApi/Auth0.AuthenticationApi.csproj

```diff
@@ -2,9 +2,10 @@
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
     <AssemblyTitle>Auth0.AuthenticationApi</AssemblyTitle>
-    <TargetFrameworks>netstandard2.0;net452;netstandard1.4;</TargetFrameworks>
     <AssemblyName>Auth0.AuthenticationApi</AssemblyName>
     <PackageId>Auth0.AuthenticationApi</PackageId>
+    <TargetFrameworks>netstandard2.0;net452;netstandard1.4;</TargetFrameworks>
+    <NetStandardImplicitPackageVersion>2.0.3</NetStandardImplicitPackageVersion>
```

Comment:
This was unfortunately required due to some internal housekeeping in System.IdentityModel.Tokens.Jwt that broke things without it in a minor.